test/python/test_nixl_api.py

```
@@ -246,7 +247,7 @@ def test_incorrect_plugin_env(monkeypatch):
         nixl_agent("bad env agent")
 
 
-def _run_xfer_telemetry_check(agent1, agent2):
+def _run_xfer_telemetry_check(agent1, agent2, expect_telemetry=True):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_run_xfer_telemetry_check`

Add return type annotation: `None`

(ANN202)